Convert metric aggs docs runtime fields (#71260)
elastic/elasticsearch: docs/reference/aggregations/metrics/rate-aggregation.asciidoc

==== Script

> If you need to run the aggregation against values not indexed you can run the
> aggregation on a <<runtime,runtime field>>. For example, if we need to adjust
> our prices before calculating rates:

Reviewer: ```suggestion
If you need to run the aggregation against values that aren't indexed, run the
aggregation on a <<runtime,runtime field>>. For example, if we need to adjust
our prices before calculating rates:
```
